app/src/main/java/ca/llamabagel/transpo/TranspoApplication.kt

```diff
@@ -5,16 +5,15 @@
 package ca.llamabagel.transpo
 
 import android.app.Application
+import androidx.appcompat.app.AppCompatDelegate
 import androidx.work.Configuration
 import androidx.work.WorkManager
 import ca.llamabagel.transpo.data.LocalMetadataSource
-import ca.llamabagel.transpo.di.DaggerComponentProvider
-import ca.llamabagel.transpo.di.CoreComponent
-import ca.llamabagel.transpo.di.DaggerCoreComponent
-import ca.llamabagel.transpo.di.SharedPreferencesModule
-import ca.llamabagel.transpo.di.InjectionWorkerFactory
+import ca.llamabagel.transpo.di.*
 import javax.inject.Inject
 
+private const val ANDROID_O_SDK = 29
```

Comment:
We don't need variables for this. Simply using `29` will do.
```suggestion

```

Reply:
it's a magic number

Reply:
We're still going to use integer literals from here on out

Reply:
so suppress the magic number warning?

Reply:
In this case, yes